src/shared/utils/dateTimeUtils.ts

```diff
@@ -42,3 +42,13 @@ export function addDurationToDate(
     }
   }
 }
+
+// checks whether the passed date is ISO format
+export function isISODate(dateString: string): boolean {
+  try {
+    const date = new Date(dateString)
+    return date.toISOString() === dateString
+  } catch (error) {
```

Comment:
not a big deal or worth fixing up since it's irrelevant, but I think if you're not going to make use of `error` you can just do:

```ts
try {
  const date = new Date(dateString)
  return date.toISOString() === dateString
} catch {
  return false
}
```

Reply:
Ahh! I tried leaving it like this: 
```
} catch() {
  return false
}
```
and it didn't work, I see, I need no parenthesis. I'll keep that in mind for the future 👍